py3-apache-beam.yaml

```diff
@@ -0,0 +1,62 @@
+# Generated from https://pypi.org/project/apache-beam/
+package:
+  name: py3-apache-beam
+  version: 2.51.0
+  epoch: 0
+  description: Apache Beam SDK for Python
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - py3-crcmod
+      - py3-orjson
+      - py3-dill
+      - py3-cloudpickle
+      - py3-fastavro
+      - py3-fasteners
+      - py3-grpcio
+      - py3-hdfs
+      - py3-httplib2
+      - numpy
+      - py3-objsize
+      - py3-pymongo
+      - py3-proto-plus
+      - py3-protobuf
+      - py3-pydot
+      - py3-python-dateutil
+      - py3-pytz
+      - py3-regex
+      - py3-requests
+      - py3-typing-extensions
+      - py3-zstandard
+      - pyarrow
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 2303bc6030072b033a567abb8c20de1d00faa86a69e2b168a952c3b692af97a2
+      uri: https://files.pythonhosted.org/packages/89/d5/48ca435cac3d73a422bcb1f51ffdeebbcee6b89107ebc5b0c3cc2f35857a/apache-beam-${{package.version}}.zip
+      extract: false
+
+  - runs: unzip apache-beam-${{package.version}}.zip
+
+  - working-directory: /home/build/apache-beam-${{package.version}}
```

Comment:
Might be worth doing `unzip -j` if you wish to skip the `working-directory` directive.